config/proco.yml

```diff
@@ -0,0 +1,17 @@
+# PURL configuration for http://purl.obolibrary.org/obo/proco
+
+idspace: PROCO
+base_url: /obo/proco
+
+products:
+- proco.owl: https://raw.githubusercontent.com/proco-ontology/PROCO/v2022-04-11/proco.owl 
+
+term_browser: ontobee
+example_terms:
+- PROCO_0000001
+
+entries:
+- exact: /v2022-04-11/proco.owl 
+  replacement: https://raw.githubusercontent.com/proco-ontology/PROCO/v2022-04-11/proco.owl 
```

Comment:
If you use the usual standard in OBO, you should use

```
http://purl.obolibrary.org/obo/proco/releases/2022-04-11/proco.owl
```
or

```
http://purl.obolibrary.org/obo/proco/2022-04-11/proco.owl
```

without the `v`.

The `v` then comes in here:

https://github.com/OBOFoundry/purl.obolibrary.org/blob/master/config/aism.yml#L17

If you use the first version IRI, with the `releases` in the IRI, then you can use a single rewrite rule like in the link, and it will work forever. Otherwise you will have to keep updating your config file!